Mathlib/Combinatorics/SimpleGraph/Copy.lean

```diff
@@ -1,43 +1,64 @@
 /-
-Copyright (c) 2025 Mitchell Horner. All rights reserved.
+Copyright (c) 2023 Yaël Dillies, Mitchell Horner. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Mitchell Horner
+Authors: Yaël Dillies, Mitchell Horner
 -/
 import Mathlib.Combinatorics.SimpleGraph.Subgraph
 
 /-!
-# Copies of Simple Graphs
+# Containment of graphs
 
 This file introduces the concept of one simple graph containing a copy of another.
 
-## Main definitions
+For two simple graph `G` and `H`, a *copy* of `G` in `H` is a (not necessarily induced) subgraph of
+`H` isomorphic to `G`.
 
-* `SimpleGraph.Copy A B` is the type of copies of `A` in `B`, implemented as the subtype of
-  *injective* homomorphisms.
+If there exists a copy of `G` in `H`, we say that `H` *contains* `G`. This is equivalent to saying
+that there is an injective graph homomorphism `G → H` them (this is **not** the same as a graph
+embedding, as we do not require the subgraph to be induced).
+
+If there exists an induced copy of `G` in `H`, we say that `H` *inducingly contains* `G`. This is
+equivalent to saying that there is a graph embedding `G ↪ H`.
 
-* `SimpleGraph.IsContained A B`, `A ⊑ B` is the relation that `B` contains a copy of `A`, that
-  is, the type of copies of `A` in `B` is nonempty. This is equivalent to the existence of an
-  isomorphism from `A` to a subgraph of `B`.
+## Main declarations
 
+Containment:
+* `SimpleGraph.Copy G H` is the type of copies of `G` in `H`, implemented as the subtype of
+  *injective* homomorphisms.
+* `SimpleGraph.IsContained G H`, `G ⊑ H` is the relation that `H` contains a copy of `G`, that
+  is, the type of copies of `G` in `H` is nonempty. This is equivalent to the existence of an
+  isomorphism from `G` to a subgraph of `H`.
   This is similar to `SimpleGraph.IsSubgraph` except that the simple graphs here need not have the
   same underlying vertex type.
+* `SimpleGraph.Free` is the predicate that `H` is `G`-free, that is, `H` does not contain a copy of
+  `G`. This is the negation of `SimpleGraph.IsContained` implemented for convenience.
 
-* `SimpleGraph.Free` is the predicate that `B` is `A`-free, that is, `B` does not contain a copy of
-  `A`. This is the negation of `SimpleGraph.IsContained` implemented for convenience.
+Induced containment:
+* Induced copies of `G` inside `H` are already defined as `G ↪g H`.
+* `SimpleGraph.IsIndContained G H` : `G` is contained as an induced subgraph in `H`.
 
 ## Notation
 
 The following notation is declared in locale `SimpleGraph`:
-
 * `G ⊑ H` for `SimpleGraph.IsContained G H`.
+* `G ⊴ H` for `SimpleGraph.IsIndContained G H`.
 -/
 
 open Finset Function
+open Fintype (card)
 
 namespace SimpleGraph
 variable {V W X α β γ : Type*} {G G₁ G₂ G₃ : SimpleGraph V} {H : SimpleGraph W} {I : SimpleGraph X}
   {A : SimpleGraph α} {B : SimpleGraph β} {C : SimpleGraph γ}
 
+/-!
+### Copies
+
+Note that induced copies of `G` inside `H` are already defined as `G ↪g H`.
```

Comment:
The induced version of this section docstring has more detail but this one basically just tells me "I am not induced", can we add a bit more here?

Reply:
I don't think I do, I'm specifically complaining about the one about "Copies" (this one) being less useful than the ones about "Containment" and "Induced containment"

Reply:
Ah, but this section is both about induced and non-induced copies, with the comment explaining that the induced stuff is already done somewhere else

Reply:
I agree that's what it was doing, and my request was for it to be doing more than that :)